delivery_automatic_package/models/delivery_carrier.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class DeliveryCarrier(models.Model):
+
+    _inherit = "delivery.carrier"
+
+    automatic_package_creation_at_delivery = fields.Boolean(
```

Comment:
Nitpicking
Python code makes things automatic, then `automatic`  have no high value here and make field name longer useless.
```suggestion
    package_creation_at_delivery = fields.Boolean(
```

Reply:
Yes, but that reflects a change from standard behavior.

Apart that, are you ok with the change ?